pallets/task/src/tests.rs

```diff
@@ -776,16 +789,25 @@ fn delete_task_after_deadline() {
 		run_to_block(1);
 		// Profile is necessary for task creation
 		assert_ok!(Profile::create_profile(Origin::signed(1), username(), interests(), HOURS));
+		let signer_balance = Balances::balance(&1);
 
 		// Ensure new task can be created.
 		assert_ok!(Task::create_task(Origin::signed(1), title(), spec(), BUDGET, get_deadline(), attachments(), keywords()));
```

Comment:
In both tests, you are only calling create_task extrinsic, while you have added functionality to update_task. Please test update_task also, with different variations of budgets. 

The rest should be good.